app/controllers/api/v1/session.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'barong/security/access_token'
+
+module API
+  module V1
+    class Session < Grape::API
+      desc 'Session related routes'
+      resource :session do
+        post '/create' do
+          acc = Account.find_by(email: params[:email])
+          return error!('401 Unauthorized', 401) unless acc
+
+          app = Doorkeeper::Application.find_by(uid: params[:application_id])
```

Comment:
API lucks documentation and validation for params. Please, don't forget to validate email using grape validators.